gemrb/plugins/AREImporter/AREImporter.cpp

```diff
@@ -1448,9 +1436,8 @@ Map* AREImporter::GetMap(const ResRef& resRef, bool day_or_night)
 		str->ReadWord(TrapSize);
 		str->ReadWord(ProID);
 		str->ReadDword(Ticks);  //actually, delaycount/repetitioncount
-		str->ReadWord(X);
-		str->ReadWord(Y);
-		str->ReadWord(Z);
+		str->ReadPoint(point);
+		str->Seek(2, GEM_CURRENT_POS); // unknown/unused 'Z'
```

Comment:
this is likely Projectile::ZPos

Reply:
yeah, I had the same inkling, but the field is private, and we seem to restrain it to these preset values:
https://github.com/gemrb/gemrb/blob/138886d702265939d96b5f6b4ba5a5c8215d5915/gemrb/core/Projectile.h#L47-L52

so its something out of scope to fix here IMO